### DISCUSSION TITLE
LeoCad crashed - dmp file

### DISCUSSION THREAD

**Discussion by JeannetteMerino** [leozide/leocad#689] [General]
Hello!
I have a dmp file. I have a crashed when trying to work. I don´t know where to send the file so I´m adding here.
Would you help me?

Thanks

[leocad.zip](https://github.com/leozide/leocad/files/6232447/leocad.zip)

**Comment by leozide** [comment]
Is this from one of the continuous builds? Can you tell me the version from the About Dialog?

**Reply by JeannetteMerino** [reply]
Hello
I was trying with all the kind of pieces and notice that that error only happend when I'm in "Other" and with the pieces that show you in the word file attached.
I don't know where to identify the About Dialog, could you tell me where can i find it?

Thanks
[error.zip](https://github.com/leozide/leocad/files/6252822/error.zip)

**Reply by leozide** [reply]
I tried an older build and I got the same crash, it's an old bug and has been fixed already. You can download the latest version from https://github.com/leozide/leocad/releases/latest and it won't happen anymore.

**Reply by JeannetteMerino** [reply]
Thanks!
